packages/redirect/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "automattic/jetpack-redirect",
+	"description": "Utilities to build URLs to the jetpack.com/redirect service",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"automattic/jetpack-constants": "@dev"
```

Comment:
Do we need this?

Reply:
No. https://github.com/Automattic/jetpack/pull/15434/commits/82c716d547818b75ca0c9dcf612fe06419dafb45